packages/room/api/fetcher.js

```diff
@@ -1,18 +1,3 @@
-import nodeFetch, {
-  Headers as NodeHeaders,
-  Request as NodeRequest,
-  Response as NodeResponse,
-} from 'node-fetch'
-
-if (!('fetch' in globalThis)) {
-  Object.assign(globalThis, {
-    fetch: nodeFetch,
-    Headers: NodeHeaders,
-    Request: NodeRequest,
-    Response: NodeResponse,
-  })
-}
-
```

Comment:
We will only use standard fetch API and no longer need node-fetch because:
1. Most ssr framework already has support for fetch API in the server
2. node 21 already has [stable fetch API](https://nodejs.org/en/blog/announcements/v21-release-announce)